mula/poetry.lock

```diff
@@ -23,38 +23,38 @@ tz = ["python-dateutil"]
 
 [[package]]
 name = "annotated-types"
-version = "0.5.0"
+version = "0.6.0"
 description = "Reusable constraint types to use with typing.Annotated"
 optional = false
-python-versions = ">=3.7"
+python-versions = ">=3.8"
```

Comment:
Does the 3.8 requirement mean we also don't support 3.7 anymore? And if so, does this mean we run into problems with hosting for blue team?